platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/schedulers/builders/TaskSchedulerConfiguration.java

```diff
@@ -51,6 +51,20 @@ public record TaskSchedulerConfiguration(
     public static final TaskSchedulerConfiguration NO_OP_CONFIGURATION =
             new TaskSchedulerConfiguration(TaskSchedulerType.NO_OP, 0L, false, false, false, false);
 
+    /**
+     * This configuration is for a simple direct task scheduler. It is not necessary to use this constant for a direct
+     * task scheduler, but it is provided for convenience.
+     */
+    public static final TaskSchedulerConfiguration DIRECT_CONFIGURATION =
+            new TaskSchedulerConfiguration(TaskSchedulerType.DIRECT, 0L, false, false, false, false);
+
+    /**
+     * This configuration is for a thread-safe direct task scheduler. It is not necessary to use this constant for a
+     * thread-safe direct task scheduler, but it is provided for convenience.
+     */
+    public static final TaskSchedulerConfiguration DIRECT_THREADSAFE_CONFIGURATION =
```

Comment:
The javadoc is helpful, thanks for adding it for context. Are there any upsides to using this convenience? If not, i think it's easier to just stick strongly to one or the other - use this field or instantiate on site - but not both.

Reply:
No upsides, it's just a convenience method. There are a few places I intend to use this constant. I added it in this PR since I was adding the other constant, but didn't think it appropriate to include the other changes within this set of changes.